example/README.md

```diff
@@ -0,0 +1,171 @@
+# Wattsight API python library usage example
+An introductory example of using Wattsight API python library for searching, fetching, and plotting Wattsight data series. A number of use cases is presented in order to introduce different data structures.
+
+## Dependencies
+
+[Wattsight API python library](https://github.com/wattsight/wapi-python/)   
+[Jupyter (iPython) Notebooks](http://jupyter.org/)   
+[Pandas Data Analysis library](http://pandas.pydata.org/)   
+[Plotly Charting library](https://plot.ly/)   
+[World Timezone Definitions for Python](http://pytz.sourceforge.net/)   
+
+## Installation of Wattsight API python library
+
+1. Using curl
+
+	```command
+	curl -LOk https://github.com/wattsight/wapi-python/archive/wapi-python-0.0.7.zip
+	unzip wapi-python-0.0.7.zip
+	cd wapi-python-wapi-python-0.0.7
+	python setup.py install
+	```
+
+2. Using pip
+
+	```command
+	pip install git+https://github.com/wattsight/wapi-python@wapi-python-0.0.7
+	```
+
+## Implementation
+Complete working code can be found in the attached Jupyter Notebook file.
+
+1. Established a session using client cridentials. 
```

Comment:
This needs some sort of description, took me a while to realise that the numbers refer to "the things we will do".

Also this one should be "Establish a session using client credentials."